modules/exploits/linux/http/linear_emerge_unauth_rce_cve_2019_7256.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/stopwatch'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  # Default root password on a Linear eMerge E3 access controller
+  ROOT_PWD = 'davestyle'.freeze
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Linear eMerge E3 Access Controller Command Injection',
+        'Description' => %q{
+          This module exploits a command injection vulnerability in the Linear eMerge
+          E3 Access Controller. The issue is triggered by an unsanitized exec() PHP
+          function allowing arbitrary command execution with root privileges.
```

Comment:
Should mention the firmware version here just in case it does get patched? Also same point r.e E3 being a range of products and not a single product.

Reply:
Also no mention of the vulnerable page here, or the vulnerable parameters either which is something that should be mentioned here.

Reply:
Done. See https://github.com/rapid7/metasploit-framework/pull/17312/commits/008ba5334b2aeea230933567d1b7503232a016b5